com.unity.mobile.notifications/Documentation~/iOS.md

```diff
@@ -0,0 +1,225 @@
+
+```c#
+iOSNotificationCenter.OnRemoteNotificationReceived += remoteNotification =>
+{
+    // When a remote notification is received, modify its contents and show it after 1 second.
+    var timeTrigger = new iOSNotificationTimeIntervalTrigger()
+    {
+        TimeInterval = new TimeSpan(0, 0, 1),
+        Repeats = false
+    };
+    
+    iOSNotification notification = new iOSNotification()
+    {
+        Title = "Remote: " + remoteNotification.Title,
+        Body = "Remote: " + remoteNotification.Body,
+        Subtitle = "Remote: " + remoteNotification.Subtitle,
+        ShowInForeground = true,
+        ForegroundPresentationOption = PresentationOption.Sound | PresentationOption.Alert,
+        CategoryIdentifier = remoteNotification.CategoryIdentifier,
+        ThreadIdentifier = remoteNotification.ThreadIdentifier,
+        Trigger = timeTrigger,
+    };
+    iOSNotificationCenter.ScheduleNotification(notification);
+};
+```
+
+### Store and retrieve custom data
+
+You can store arbitrary string data on the notification with [iOSNotification.Data](../api/Unity.Notifications.iOS.iOSNotification.html#Unity_Notifications_iOS_iOSNotification_Data), and retrieve it later from the received notification.
+
+```c#
+var notification = new iOSNotification();
+notification.Data = "{\"title\": \"Notification 1\", \"data\": \"200\"}";
+iOSNotificationCenter.ScheduleNotification(notification);
+```
+
+The following code example shows how to retrieve the last notification the app received.
+
+```c#
+var notification = iOSNotificationCenter.GetLastRespondedNotification();
+if (notification != null)
+{
+    var msg = "Last Received Notification: " + notification.Identifier;
+    msg += "\n - Notification received: ";
+    msg += "\n - .Title: " + notification.Title;
+    msg += "\n - .Badge: " + notification.Badge;
+    msg += "\n - .Body: " + notification.Body;
+    msg += "\n - .CategoryIdentifier: " + notification.CategoryIdentifier;
+    msg += "\n - .Subtitle: " + notification.Subtitle;
+    msg += "\n - .Data: " + notification.Data;
+    Debug.Log(msg);
+}
+```
+
+If the user opens the app from a notification, [iOSNotificationCenter.GetLastRespondedNotification](../api/Unity.Notifications.iOS.iOSNotificationCenter.html#Unity_Notifications_iOS_iOSNotificationCenter_GetLastRespondedNotification) also returns that notification. Otherwise it returns null.
+
+#### Set custom data for remote notifications
+
+Sometimes users might want to set custom data on the payload for a remote notification and retrieve it by [iOSNotification.Data](../api/Unity.Notifications.iOS.iOSNotification.html#Unity_Notifications_iOS_iOSNotification_Data). Below is an example of setting a string as `data` on the payload.
```

Comment:
```suggestion
Sometimes, you might want to set custom data on the payload for a remote notification and retrieve it using [iOSNotification.Data](../api/Unity.Notifications.iOS.iOSNotification.html#Unity_Notifications_iOS_iOSNotification_Data). The example below shows how to set a string as `data` on the payload.
```

Reply:
I assumed "users" here refers to developers, i.e. people who use this package, in which case we need to write this in second-person ("you"). "Users" in the context of these package docs refers to end-users.

Reply:
> I assumed "users" here refers to developers